controllers/weather.js

```diff
@@ -1,11 +1,19 @@
 const { fetchWeatherDataByCity } = require('../services/weather');
 
-// @desc     Get weather data
+// @desc     Get weather data for index page (duplicate with getCityWeather, should be refactor or remove)
 // @router  GET /weather/index
 // @access  public
 module.exports.getWeatherIndex = async (req, res, next) =>
 {
     const city = req.params.city;
+
+    if (!city || "" === city)
```

Comment:
cai nay chi can `if(!city)` la du roi Cuong oi

Reply:
ong co the tham khao coding style o day nhe: https://github.com/airbnb/javascript.